registry/config.go

```diff
@@ -143,6 +143,13 @@ func (config *serviceConfig) LoadInsecureRegistries(registries []string) error {
 
 skip:
 	for _, r := range registries {
+		if strings.Contains(r, "://") {
+			// Insecure registry does not contain a scheme.
+			// before returning err, roll back to original data
+			config.ServiceConfig.InsecureRegistryCIDRs = originalCIDRs
+			config.ServiceConfig.IndexConfigs = originalIndexInfos
```

Comment:
hm, this "set new values, then revert if it didn't validate" is a bit awkward (I know it was there before), looks like we need to clean this up a bit in a follow up